Tests/ReactiveSwiftTests/SignalProducerLiftingSpec.swift

```diff
@@ -1666,5 +1666,47 @@ class SignalProducerLiftingSpec: QuickSpec {
 				expect(latestValues?.1) == 2
 			}
 		}
+		
+		describe("groupBy") {
```

Comment:
yeah right, i've split this up into 2 separated cases and extended them to check the lifetime of the inner producers